libafl/src/executors/forkserver.rs

```diff
@@ -712,10 +718,11 @@ impl<'a> ForkserverExecutorBuilder<'a, StdShMemProvider> {
     }
 
     #[must_use]
-    /// Place the input at this position, this is equivalent to putting "@@" here.
-    pub fn arg_input_file(mut self) -> Self {
-        self.arguments.push(OsString::from("@@"));
-        self
+    /// Place the input at this position and set the filename for the input.
+    pub fn arg_input_file<P: AsRef<Path>>(self, path: P) -> Self {
```

Comment:
This also needs to push the filename as `arg`, doesn't it?

Reply:
Like, my idea (and correct me if you don't like it) would be that you can do 
`builder().program("file").arg_input_file("myfile").build()`

Reply:
`self.arg(path.as_ref());` already pushs the filename into self.arguments (?)

Reply:
Right